src/i18n/en/Payroll.PayrollOverview.json

```diff
@@ -1,4 +1,5 @@
 {
+  "backButton": "Back",
```

Comment:
for my learning: I'm guessing there was a discussion at some point about whether it would be valuable to have a "common" dictionary for recurring simple copies like "back", "continue", etc. 

or is it the i18next way of doing things to have every copy colocated to its own respective dictionary per component?